commands.js

```diff
@@ -2910,8 +2910,11 @@ exports.commands = {
 				if (i === 'global') continue;
 				let targetRoom = Rooms.get(i);
 				if (!targetRoom) continue; // shouldn't happen
-				if (targetRoom.isPrivate && !(i in user.games)) continue;
 				let roomData = {};
+				if (targetRoom.isPrivate) {
+					if (!(i in user.roomCount)) continue;
+					roomData.isPrivate = true;
+				}
```

Comment:
Also consider that the reason we do `user.games` in the first place is that the user might be in the game but not the room (if they've disconnected, for instance).